views/includes/metaHead.njk

```diff
@@ -9,11 +9,6 @@
         isNotProduction: {{ appData.isNotProduction }},
         locale: '{{ locale }}',
         localePrefix: '{{ localePrefix }}',
-        isModernBrowser:
```

Comment:
We don't use this check or the corresponding CSS class anywhere so I've removed the check.